chunky/src/java/se/llbit/chunky/renderer/postprocessing/AldridgeFilmicFilter.java

```diff
@@ -0,0 +1,88 @@
+package se.llbit.chunky.renderer.postprocessing;
```

Comment:
I like the tonemap changes; they're also a good candidate to be separated into a separate PR.